light-zk.js/tests/utxo.test.ts

```diff
@@ -319,11 +261,12 @@ describe("Utxo Functional", () => {
       verifierProgramLookupTable:
         lightProvider.lookUpTables.verifierProgramLookupTable,
     });
-    if (utxo3) {
+    if (typeof utxo3 !== "boolean") {
```

Comment:
is this going to work without the changes in your other pr?

Reply:
Yes it is, but anyway I removed it to make things clearer.